public/app/core/components/PermissionList/AddPermission.tsx

```diff
@@ -88,56 +86,52 @@ class AddPermissions extends Component<Props, NewDashboardAclItem> {
     const newItem = this.state;
     const pickerClassName = 'min-width-20';
     const isValid = this.isValid();
+
     return (
-      <div className="gf-form-inline cta-form">
+      <div className="cta-form">
         <button className="cta-form__close btn btn-transparent" onClick={onCancel}>
           <Icon name="times" />
         </button>
-        <form name="addPermission" onSubmit={this.onSubmit}>
-          <h5>Add Permission For</h5>
-          <div className="gf-form-inline">
-            <div className="gf-form">
-              <div className="gf-form-select-wrapper">
-                <select className="gf-form-input gf-size-auto" value={newItem.type} onChange={this.onTypeChanged}>
-                  {dashboardAclTargets.map((option, idx) => {
-                    return (
-                      <option key={idx} value={option.value}>
-                        {option.text}
-                      </option>
-                    );
-                  })}
-                </select>
-              </div>
-            </div>
-
-            {newItem.type === AclTarget.User ? (
-              <div className="gf-form">
-                <UserPicker onSelected={this.onUserSelected} className={pickerClassName} />
-              </div>
-            ) : null}
-
-            {newItem.type === AclTarget.Team ? (
-              <div className="gf-form">
-                <TeamPicker onSelected={this.onTeamSelected} className={pickerClassName} />
-              </div>
-            ) : null}
-
-            <div className="gf-form">
-              <Select
-                isSearchable={false}
-                options={dashboardPermissionLevels}
-                onChange={this.onPermissionChanged}
-                className="gf-form-select-box__control--menu-right"
-              />
-            </div>
-
-            <div className="gf-form">
-              <button data-save-permission className="btn btn-primary" type="submit" disabled={!isValid}>
+        <h5>Add Permission For</h5>
+        <Form maxWidth="none" onSubmit={this.onSubmit}>
+          {() => (
+            <InlineFieldRow>
+              <InlineField>
+                <Select
+                  isSearchable={false}
+                  value={this.state.type}
+                  options={dashboardAclTargets}
+                  onChange={this.onTypeChanged}
+                />
+              </InlineField>
+
+              {newItem.type === AclTarget.User ? (
+                <InlineField>
+                  <UserPicker onSelected={this.onUserSelected} className={pickerClassName} />
+                </InlineField>
+              ) : null}
+
+              {newItem.type === AclTarget.Team ? (
+                <InlineField>
+                  <TeamPicker onSelected={this.onTeamSelected} className={pickerClassName} />
+                </InlineField>
+              ) : null}
+
```

Comment:
A label here that said "Can"  would be nice, to make it similar to table and not just be 3 selects in a row